pkg/health/server.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package health
+
+import (
+	"context"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/pkg/eestream"
+	"storj.io/storj/pkg/overlay"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb"
+	"storj.io/storj/pkg/storj"
+)
+
+var (
+	mon = monkit.Package()
+	// Error wraps errors returned from Server struct methods
+	Error = errs.Class("HealthEndpoint error")
+)
+
+type HealthEndpoint struct {
```

Comment:
Just `Endpoint`, because the package name already is `health`. Otherwise the full name of the type will be `health.HealthEndpoint`.

Reply:
done